server/views/error.jade

```diff
@@ -0,0 +1,6 @@
+extends layout
```

Comment:
@RonLek Since we are not using the backend for SSR, we won't need the views folder.